user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -210,3 +299,45 @@ bool Timepix3RawEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::S
 
   return data_found;
 }
+
+void Timepix3RawEvent2StdEventConverter::loadCalibration(std::string path, char delim, std::vector<std::vector<float>>& dat) const {
+    // copied from Corryvreckan EventLoaderTimepix3
+    std::ifstream f;
+    f.open(path);
+    dat.clear();
+
+    // check if file is open
+    if(!f.is_open()) {
+        EUDAQ_WARN("Cannot open input file:\n\t" + path);
+        // throw InvalidValueError(config_, "calibration_path", "Parsing error in calibration file.");
```

Comment:
Shouldn't this be an error? We can nicely throw stuff now... If I supply a calibration file I expect this to not silently fail but to tell me if something went wrong.

(same might apply to CPX2 calibration)

Reply:
Yes, you're right. Then should I implement an exception `ConfigInvalid()` as `DataInvalid()` doesn't sound quite right here...

Reply:
Currently we'll only catch the former:

https://gitlab.cern.ch/corryvreckan/corryvreckan/-/blob/master/src/modules/EventLoaderEUDAQ2/EventLoaderEUDAQ2.cpp#L499

I would like to not overcomplicate this if possible. ;)

Reply:
Ok, sounds good, then I'll throw a `DataInvalid()`. (thumbs_up)